### PR TITLE
Fix mistakes in the Gruntz algorithm example

### PR DESCRIPTION
**Pull request by certik**
1. how to choose subexpression omega (this part was: wrong)
2. the rewrite stage (was: non-exsistent), I hope
   now it's clear how to rewrite elements of the mrv set,
   and this is not a magic step for our readers (or worse,
   for "authors" of the SymPy) anymore.
3. mention the rewrite stage in comment about the
   algorithm termination.
4. some minor fixes, i.e. "The goal is to calculate" was
   dropped, the Gruntz algorithm IS about calculation limits
   at the infinity.

### CODE REVIEW

File: gruntz.tex [hunk 1]
```diff
@@ -70,7 +95,7 @@
     \to 2 + {1\over x}
 \end{equation}
 
-Since the result ($2+{1\over x}$) still depends on $x$, the above procedure is
+In our example the result ($2+{1\over x}$) still depends on $x$, so the above procedure is
```
Comment (certik) [inline gruntz.tex]: @asmeurer recommended to use the passive form, so no `our`.

File: gruntz.tex [hunk 2]
```diff
@@ -48,14 +47,40 @@
...
+c = \lim\limits_{x\to\infty} \frac{\log{f(x)}}{\log{g(x)}},
+\qquad
+A = e^{\log f - c \log g}
+\end{equation}
+Note that this step include calculation of more simple limits, for instance
+\begin{equation}
+	\lim\limits_{x\to\infty} \frac{\log{e^{x + 2 e^{-x}}}}{\log e^{-x}}=
+	\lim\limits_{x\to\infty} \frac{x + 2 e^{-x}}{-x} = -1
+\end{equation}
+In our example we obtain the rewritten mrv set: $\{{1\over\omega},
```
Comment (certik) [inline gruntz.tex]: Use a passive form, no `our`.

File: gruntz.tex [hunk 3]
```diff
@@ -48,14 +47,40 @@
...
+\omega, {1\over\omega}e^{2\omega}\}$. This can be done in the SymPy with
```
Comment (certik) [inline gruntz.tex]: Probably `the` should not be there, though I am not 100% sure.

**Comment by certik** [comment]
Left some simple comments, otherwise I think this looks good. Thanks for filling in the details about how to rewrite the expressions, I originally skipped it, but it's nice to have this in.

File: gruntz.tex [hunk 4]
```diff
@@ -48,14 +47,40 @@
 dict_keys([exp(x + 2*exp(-x)), exp(x), exp(-x)])
 \end{verbatim}
 
-Next, an arbitrary item $\omega$ is taken from mrv that converges to zero for
-$x\to\infty$. The item $\omega=e^{-x}$ is obtained. If such a term is not
+Next, an arbitrary item $\omega$ is taken from mrv set that converges to zero for
+$x\to\infty$ and doesn't have subexpressions in the given mrv set.  In our case,
```
Comment (certik) [inline gruntz.tex]: Use passive form, remove `our`.

File: gruntz.tex (hunk 3 above)
Comment (asmeurer) [inline gruntz.tex]: Yeah, "the SymPy" is not correct.

File: gruntz.tex [hunk 5]
```diff
@@ -48,14 +47,40 @@
...
+>>> m = mrv(exp(x+2*exp(-x))-exp(x) + 1/x, x)
+>>> w = Symbol('w')
+>>> rewrite(m[1], m[0], x, w)[0]
+     2*w
+1   e      1
+- + ---- - -
+x    w     w
+\end{verbatim}
+Then the rewritten subexpressions are
+substituted back into $f(x)$ in~(\ref{gruntz_example_fn})
```
Comment (asmeurer) [inline gruntz.tex:79]: The parentheses should be unnecessary here.

**Pull request merged**